### PR TITLE
Detect builtins used

### PR DESCRIPTION
**Pull request by fridex**

### CODE REVIEW

**Comment by fridex** [comment]
PTAL, ready to be revied.

File: invectio/cli.py [hunk 1]
```diff
@@ -93,20 +96,28 @@ def cli(ctx=None, verbose: bool = False) -> None:
...
+    is_flag=True,
+    show_default=True,
     help="Do not report usage of Python's builtins.",
 )
 def whatuses(
     path: str,
     ignore_errors: bool = False,
     without_standard_imports: bool = False,
     without_builtin_imports: bool = False,
+    without_builtins: bool = False,
```
Comment (KPostOffice) [inline invectio/cli.py:112]: Why is the default False here and True inside of the InvectioLibraryUsageVisitor?

**Review by KPostOffice — COMMENTED** [review]
looks good. I have one comment

File: invectio/cli.py (hunk 1 above)
Comment (fridex) [reply invectio/cli.py:112]: Nice catch! Unified this logic in 7217eeb. PTAL 👍🏻

**Pull request merged**